website/js/components/designer/menu-testall.vue

```diff
@@ -149,9 +149,11 @@ data:function(){
           try{
               const now = new Date();
               const usableFilename = ( (this.filename && this.filename.length > 0) ? this.filename : 'TestAll');
+              const token = window.sessionStorage.getItem('id_token');
```

Comment:
Hi Taruk, The id token is stored in the browser local storage in website/js/lib/store/user/actions.js  line 33.   Here I am retrieving it from local storage so i can pass it to the API that invokes the TESTALL lambda, so that it in turn can set the lex session attribute required to allow TESTALL to work when QnABot bot authentication is required.